tests/test_bulk_rx_basic.py

```diff
@@ -3,11 +3,21 @@
 # This Software is subject to the terms of the XMOS Public Licence: Version 1.
 
 import random
-import xmostest
 from  usb_packet import *
 import usb_packet
 from usb_clock import Clock
-from helpers import do_usb_test, runall_rx
+from helpers import do_usb_test, get_usb_clk_phy
+from usb_clock import Clock
+from usb_phy_utmi import UsbPhyUtmi
+import Pyxsim
+import pytest
+import os
+
+
+ARCHITECTURE_CHOICES = ['xs2', 'xs3']
```

Comment:
This seems to be duplicated from the previous test file, and in all of the files. Unless I'm missing something, I think this information should be written into one file, and included where needed, otherwise maintaining this is going to be difficult as the project evolves.

Reply:
Moved the duplicate to helpers.py. Thanks.